qhub/template/stages/07-kubernetes-services/jupyterhub.tf

```diff
@@ -107,6 +107,11 @@ module "jupyterhub" {
       name = "dask-etc"
       namespace = var.environment
       kind = "configmap"
+    },
```

Comment:
I'd prefer if extra mounts was instead added within the module so that we can directly reference the config map name. There is a terraform `concat` function that can be used.